tools/manual_typings/project/document.d.ts

```diff
@@ -0,0 +1,3 @@
+interface Document {
+  elementsFromPoint(x: number, y: number): Element[];
```

Comment:
Nice, I like the new DOM API usage here.

Reply:
![Obama - brush your shoulders off](http://stream1.gifsoup.com/view2/2287996/obama-swag-o.gif)